lib/trim-emoji.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const emojiRegex = require('emoji-regex');
+
+module.exports = text => {
+	let match;
+	let regex = emojiRegex();
+	while ((match = regex.exec(text))) {
+		const emoji = match[0];
+		const codePoints = [...emoji].length;
+		if (match.index === 0) {
+			text = text.substr(codePoints).trim();
```

Comment:
I would prefer using `slice` instead of `substr`.

Reply:
Sure. But sadly you rejected this feature, https://github.com/sindresorhus/awesome-lint/issues/60#issuecomment-478368459.